go.mod

```diff
@@ -105,11 +109,16 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 	k8s.io/klog/v2 v2.30.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65 // indirect
+	kubevirt.io/containerized-data-importer-api v1.41.0 // indirect
+	kubevirt.io/controller-lifecycle-operator-sdk v0.2.1 // indirect
 	sigs.k8s.io/json v0.0.0-20211020170558-c049b76a60c6 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.0 // indirect
 )
 
 replace (
+	kubevirt.io/containerized-data-importer-api => github.com/kubevirt/containerized-data-importer-api v1.41.1-0.20211201033752-05520fb9f18d
 	sigs.k8s.io/apiserver-network-proxy/konnectivity-client => sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.24
 	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.0.0
 )
+
+replace sigs.k8s.io/cluster-api-provider-kubevirt => github.com/nirarg/cluster-api-provider-kubevirt-1 v0.0.0-20211212090220-ea93f714afd5
```

Comment:
why is this coming from your personal fork?

Reply:
As the capk repository is still in work in progress, not all changes are merged
Once all the changes from my local branch will get merged to openshift, will change it to use it

Reply:
Changed to use the fork under openshift